tests/modeling/test_estimation_steps.py

```diff
@@ -88,14 +88,14 @@ def test_remove_estimation_step(testdata, load_model_for_test):
 def test_add_covariance_step(testdata, load_model_for_test):
```

Comment:
In this test I would also add a test for at least one of the new options.

Reply:
Added for MATRIX=S